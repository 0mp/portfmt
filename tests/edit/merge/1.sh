echo DISTVERSION=1.2.1 | ${PORTEDIT} merge 1.in | \
	diff -L 1.expected -L 1.actual -u 1.expected -
