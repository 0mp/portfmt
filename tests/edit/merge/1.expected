DISTVERSION=	1.2.1
